server/migrations/20200805094217-create-historicalValidatorStats-table.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return queryInterface.createTable('HistoricalValidatorStats', {
+      id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true }, //primary-key
+      stash_id: { type: DataTypes.STRING, allowNull: false, references: { model: 'Validators', key: 'stash' } },
+      block: { type: DataTypes.STRING, allowNull: false }, // blocknumber
+      exposure: { type: DataTypes.JSON, allowNull: false },
+      commissionPer: { type: DataTypes.FLOAT, allowNull: false },
+      preferences: { type: DataTypes.INTEGER, allowNull: false },// preferences - ValidatorPrefs
+      apr: { type: DataTypes.FLOAT, allowNull: false },
+      uptime: { type: DataTypes.STRING, allowNull: false },
+      movingAverages: { type: DataTypes.INTEGER, allowNull: false },
```

Comment:
I think we can remove this too, movingAverages is not an INTEGER either it would be a collection, but most of these values `uptime` `apr` etc are already moving averages